pkg/asset/quota/openstack/openstack.go

```diff
@@ -25,14 +41,37 @@ func Constraints(ci *validation.CloudInfo, controlPlanes []machineapi.Machine, c
 	var ret []quota.Constraint
 	ret = controlPlane(ci, ctrplConfigs)
 	ret = append(ret, compute(ci, computeReplicas, computeConfigs)...)
+	ret = append(ret, getNetworkConstraints(networkType)...)
 	return aggregate(ret)
 }
 
+func getNetworkConstraints(networkType string) (ret []quota.Constraint) {
```

Comment:
nit: you don't need a named return value because you declare it again in the `return` statements

Reply:
I'll fix that, yeah it's a leftover. Thanks!